webapp/src/widgets/valueSelector.scss

```diff
@@ -41,21 +42,32 @@
         width: 100%;
         justify-content: center;
         align-items: center;
+
         .label-container {
             flex-grow: 1;
             display: flex;
         }
+
         .MenuWrapper {
             display: flex;
             align-self: flex-end;
+
             .IconButton {
                 background-color: unset;
+
                 &:hover {
                     background-color: rgba(var(--body-color), 0.1);
                 }
             }
         }
     }
+
+    div:nth-child(3) {
```

Comment:
Note to reviewer-

These styles are hardcoded in the library and are not configurable from either props or the style function.